README.rst

```diff
@@ -182,6 +182,104 @@ RLlib Quick Start
 .. _`RLlib`: https://docs.ray.io/en/latest/rllib.html
 
 
+Ray Serve Quick Start
+---------------------
+
```

Comment:
be sure to add to toc at top of readme??

Reply:
aha missed that. thanks!